pkg/cmd/cli/describe/printer.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 var (
-	buildColumns            = []string{"NAME", "TYPE", "FROM", "STATUS", "STARTED"}
+	buildColumns            = []string{"NAME", "TYPE", "FROM", "STATUS", "STARTED", "REASON"}
```

Comment:
i assume that when you get error build/failed build you go then investigate why it is like that and you run oc describe ultimately